packages/aws-cdk/lib/api/bootstrap/bootstrap-template.yaml

```diff
@@ -212,6 +212,9 @@ Resources:
             Status: Enabled
             NoncurrentVersionExpiration:
               NoncurrentDays: 365
+          - Status: Enabled
+            AbortIncompleteMultipartUpload:
+              DaysAfterInitiation: 7
```

Comment:
The Trusted Advisor Cost Optimization S3 Incomplete Multipart Upload Abort Configuration check recommended action suggests uploads that are not completed after 24 hours are unlikely to be completed.

As the CleanUpOldVersions rule applies to all objects in the bucket the AbortIncompleteMultipartUpload can be in the same rule.

Suggest:

```yaml
          - Id: CleanupOldVersions
            Status: Enabled
            NoncurrentVersionExpiration:
              NoncurrentDays: 365
            AbortIncompleteMultipartUpload:
              DaysAfterInitiation: 1
```